device-ecosystem/README.md

```diff
@@ -0,0 +1,57 @@
+# Device Ecosystem for DRA
+
+DRA is going to stable in 1.34 of Kubernetes. As more device vendors provide support for DRA, we want to have a tracking page where people can do to see who supports DRA.
+
+## Contributing
+
+We expect the status of the DRA drivers and their corresponds operators to change over time. Please feel free to add implementations as they graduate.
+
+This document is not official supported by Kubernetes and is mainly used by #wg-device-management to track where device vendors are in their integrations for DRA.
+
+## DRA Features
+
+One area that we want to track is not just DRA availability but also support of various DRA features as they graduate.
+
+### Table
+
+| Vendor  | DRA Driver
+|---------|------------
+| NVIDIA  | [dra-driver](https://github.com/NVIDIA/k8s-dra-driver-gpu)
+| AMD     | could not find
+| Intel   | [intel-resource-drivers-for-kubernetes](https://github.com/intel/intel-resource-drivers-for-kubernetes)
+| Google (TPU)  | could not find
+
+
+### NVIDIA
+
+#### NVIDIA DRA Driver
+
+NVIDIA has the [dra-driver](https://github.com/NVIDIA/k8s-dra-driver-gpu).
+
+As of August 2025, one can install the DRA-driver via a helm chart to support NVIDIA + DRA.
+
+#### GPU Operator
```

Comment:
mmm, not sure if it needs a separate entry. It can be part of NVIDIA-dra driver as an option to install them ad part of gpu-operator

Reply:
Yea I guess this is my Openshift bias as we are really looking to support GPU use cases via GPU Operator.

I can remove the operator but I like bundling is important as this may be of interest to downstream distributions.

So even if DRA driver is provided we would want some way to bundle it without having to require admins to install their own chart. But t